### ISSUE TITLE
Updated icon for macOS Big Sur

### ISSUE THREAD

**Issue by godly-devotion** [Caldis/Mos#279]
**这个问题是否源自某个缺陷？如果是，可于此处描述一下 | Is your feature request related to a problem? Please describe.**

macOS Big Sur is updating all the system icons with a rounded-rectangle shape and a front-facing perspective. Some 3rd party developers including Adobe are converting their app icons to match up with these system icons.

![](https://developer.apple.com/design/human-interface-guidelines/macos/images/mail-photos-preferences-big-sur.png)

![](https://developer.apple.com/design/human-interface-guidelines/macos/images/preview-xcode-text-edit-big-sur.png)

**说说你的新点子 | Describe the solution you'd like**

Could we get an updated icon that follows the rounded-rectangle shape so that the icon style is consistent with others in Big Sur?

**Comment by Caldis** [comment]
You just found the reason for the next major update

**Comment by ultratiem** [comment]
![Mos](https://user-images.githubusercontent.com/2991193/88281988-80bca580-cc9d-11ea-9e4a-d27186805ee4.png)

**Comment by godly-devotion** [comment]
@ultratiem I think thats a good start and I understand you probably want to emulate a mouse by putting the tail at the end but my immediate reaction was that it looks wrong because the wire should be at the top. How about a design closer to the Apple Mighty Mouse or the Magic Mouse?

I think Mos' latest icon (with the neon light design) looks pretty cool in its own way and is unique. Maybe we can try to give it a rounded rectangle shape first to see how that works.

**Comment by ultratiem** [comment]
@godly-devotion thank you for taking the time to comment. Unfortunately I'm not looking for feedback of any kind.

**Comment by sn0wyfall** [comment]
@ultratiem Your icon looks really good! I like it. That seems like a rather passive aggressive comment though.

**Comment by Caldis** [comment]
@ultratiem Cool, did you draw this icon for Mos? Sorry I'm late, it looks very nice.

**Comment by sn0wyfall** [comment]
@ultratiem I apologize, my comment (after the first two sentences) was not directed to you. It was directly towards @godly-devotion.

**Comment by ultratiem** [comment]
> @ultratiem Cool, did you draw this icon for Mos? Sorry I'm late, it looks very nice.

Yes, I'm the artist. The background (rounded square) isn't suitable. If you're interested, I can uploaded a proper version. We would have to drop the Apple logo due to copyrights, but the rest should be fine.

@sn0wyfall no issues with your comment at all. I was replying to the person you were replying to and didn't appreciate his feedback in the least. Non of it had any merit other than "I don't like it", which isn't valuable to me at all. Hence my snarky response.

**Comment by godly-devotion** [comment]
@ultratiem I apologize if what I said seemed critical.

Mos has a really nice unique design that we probably want to keep. By just adding the rounded corners, we can actually meet the icon design of Big Sur pretty easily. Here is an example of what I mean that someone did #305

**Comment by witchope** [comment]
<img width="512" alt="mos" src="https://user-images.githubusercontent.com/25495661/103200468-5d6ee400-4928-11eb-9c9f-8f8ef7bbeaba.png">

**Comment by witchope** [comment]
<img width="512" alt="mos" src="https://user-images.githubusercontent.com/25495661/103200533-8db68280-4928-11eb-9804-36eb50cd56fd.png">

**Comment by Caldis** [comment]
> > @ultratiem Cool, did you draw this icon for Mos? Sorry I'm late, it looks very nice.
> 
> Yes, I'm the artist. The background (rounded square) isn't suitable on this one. However, if you're interested, I can uploaded a proper version. We would have to drop the Apple logo due to copyrights, but the rest should be fine.
> 
> @sn0wyfall no issues with your comment at all. I was replying to the person you were replying to and didn't appreciate his feedback in the least. Non of it had any merit other than "I don't like it", which isn't valuable to me at all. Hence my snarky response.

Wow, it's my pleasure, I really like the icons you provided above, the light and shadow effects are very Awesome. but I think this version of the icons maybe have big differences from the current ones, such as the color or the realistic style, which might be confusing for the current users. Also considering we need a status bar icon in the same style, maybe a more abstract style would be a better choice?
